.github/workflows/CI.yml

```diff
@@ -186,3 +186,50 @@ jobs:
           name: prebuilds
           pattern: prebuilds-*
           delete-merged: true
+  smoke-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macos-latest
+        pm:
+          - npm
+          - yarn
+          - pnpm
+    steps:
+      - uses: actions/setup-node@v4
+      - run: npm init --init-module testproject -y 
+
+      - if: ${{ matrix.pm == 'npm' }}
+        run: |
+          npm install "git+${{ github.repositoryUrl }}#${{ github.ref }}"
+        env:
+          NPM_CONFIG_LOGLEVEL: verbose
+
```

Comment:
It would be great to run a simple example where zeromq is imported. This will ensure the runtime compatibility via the smoke test for different Nodejs versions.

Reply:
I was thinking of possibly co-opting https://github.com/zeromq/zeromq.js/blob/ea5f35ceef48a9847562d53f839b3ffb74956711/examples/package.json